packages/blade/src/components/Input/DropdownInputTriggers/InputChevronIcon.native.tsx

```diff
@@ -14,12 +14,12 @@ const InputChevronIcon = (props: SelectChevronIconProps): React.ReactElement =>
     <StyledChevronIconContainer accessibilityLabel="Chevron Icon" onPress={props.onClick}>
       <ChevronDownIcon
         display={props.isOpen ? 'none' : 'flex'}
-        color="surface.text.normal.lowContrast"
+        color={`surface.text.${props.isDisabled ? 'placeholder' : 'normal'}.lowContrast`}
```

Comment:
nit: extract to a variable in the component body.

Reply:
done. Also moved this into separate component so it can be reused in web and native both